app/ledger.js

```diff
@@ -232,6 +232,11 @@ const doAction = (action) => {
       setTimeout(networkConnected, 1 * msecs.second)
       break
 
+    case appConstants.APP_NAVIGATOR_HANDLER_UPDATE:
```

Comment:
related to the above comment, the app constant name should match the action name